src/main/java/org/opensearch/security/securityconf/impl/v7/InternalUserV7.java

```diff
@@ -154,5 +153,11 @@ public void setStatic(boolean _static) {
             this._static = _static;
         }
         
-        
+        public String getSaved_tenant(){
+            return saved_tenant;
+        }
+
+        public void setSaved_tenant(String s){
```

Comment:
Would it be more clear to name the parameter with words like `savedTenant` in stead of a single character `s`?

Reply:
Sure; changed

Reply:
Is this taken care of?

Reply:
yes